docs/pages/access-controls/sso/azuread.mdx

```diff
@@ -1,32 +1,35 @@
 ---
-title: SSH Authentication with Active Directory (AD) on Azure
-description: How to configure SSH access with Active Directory (AD) on Azure using Teleport
-h1: SSH Authentication with Azure Active Directory (AD)
+title: Teleport Authentication with Azure Active Directory (AD)
+description: How to configure Teleport access with Active Directory (AD) on Azure.
```

Comment:
```suggestion
description: How to configure Teleport access with Azure Active Directory.
```

There are several forms of "Active Directory on Azure" - Azure AD is only one of them. I think we should be specific here.